evennia/commands/cmdhandler.py

```diff
@@ -377,63 +320,29 @@ def _get_cmdsets(obj):
                 _msg_err(caller, _ERROR_CMDSETS)
                 raise ErrorReported(raw_string)
             try:
-                returnValue((obj.cmdset.current, list(obj.cmdset.cmdset_stack)))
+                if extra:
+                    extra = yield obj.get_extra_cmdsets(caller, current, cmdsets)
+                    returnValue(extra)
```

Comment:
I think in latest Twisted, `returnValue` is not actually needed anymore, and a regular `return value` now works from inside an `@inlineCallbacks`-decorated function.. But I've not actually tested it ...